layouts/shortcodes/post-tweets.html

```diff
@@ -0,0 +1,103 @@
+<!-- Posts -->
+<div x-data="{ posts: [{},{}] }" x-init="
+fetch('https://rapidsai.github.io/site-data/posts.json')
```

Comment:
This posts.json needs to be updated before launching. The latest content here is "Announcing RAPIDS 22.10" which is not very current.

Reply:
Hmm https://github.com/rapidsai/site-data appears to have stalled. @ajschmidt8 to check.

Reply:
The job that updated that file was on a Jenkins server that has been shutdown.

If we want to manually update the file to reflect some newer content for now, we can do that.

We'll have to recreate the automated cron job in GHAs sometime soon.

Reply:
@ajschmidt8 added a solution